packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -439,6 +442,94 @@ describe('Ceramic integration', () => {
     })
   })
 
+  it('Multiquery with genesis content provided', async () => {
+    await withFleet(2, async ([ipfs1, ipfs2]) => {
+      await swarmConnect(ipfs1, ipfs2)
+      const ceramic1 = await createCeramic(ipfs1, false)
+      const ceramic2 = await createCeramic(ipfs2, false)
+
+      const content = null
+      const metadata = {
+        controllers: [ceramic1.did.id],
+        family: 'family',
+        tags: ['x', 'y'],
+      }
+
+      // Create a deterministic TileDocument
+      const stream1 = await TileDocument.create(
+        ceramic1,
+        content,
+        { ...metadata, deterministic: true },
+        { anchor: false, publish: false }
+      )
```

Comment:
Can we add a test that does the same test but without `TileDocument.create`? We can use `StreamID.fromGenesis(genesisCommit)` to get the streamId for the multiquery.